<launch>

    <node pkg="tf2_ros" type="static_transform_publisher" name="map_world"
          args="0 0 0 0 0 0 1 map world" />

    <node name="vslam2_mono" pkg="vslam2"
          type="vslam2_mono" output="screen">

        <param name="Vocab_path" type="string" value="$(find vslam2)/ORB_SLAM3/Vocabulary/ORBvoc.txt" />
        <param name="Params" type="string" value="$(find vslam2)/ORB_SLAM3_ROS/params/Monocular/NCLT.yaml" />
        <param name="Output_name" type="string" value="NCLT" />

        <param name="Visualize" type="bool" value="true" />
        <param name="Do_Rectify" type="bool" value="false" />
        <param name="Do_Equalize" type="bool" value="false" />

        <param name="pointcloud_frame_id" type="string" value="map" />
        <param name="world_frame_id" type="string" value="map" />
        <param name="camera_link" type="string" value="lb3_cam5" />

        <remap from="/camera/image_raw" to="/lb3/cam5/image_raw" />
        <remap from="/imu" to="/imu0" />

        <param name="min_observations" type="int" value="1" /> <!-- If judge_optimized, then the number of optimizations -->
        <param name="only_quality_observations" type="bool" value="false" />
        <param name="judge_optimized" type="bool" value="false" />
        <param name="min_local_points" type="int" value="1" />
    </node>

    <!--  <node type="rviz" name="rviz" pkg="rviz" args="-d $(find vslam2)/ORB_SLAM3_ROS/rviz/rviz_vio_mono.rviz" />-->

</launch>